Meden-Agan-27/travel-worms
# Style trials

what I did:
- profile page ('My Profile')
- counter in bookshelf show
- duplicate profile card removed
- path to profile page fixed
How to test:
- log in and play with your profile (edit, add picture, delete)
- play with a bookshelf (add books, see if counters works good)
- from different pages, click 'My Profile' from menu to see if it is working well
